ppt/PPTConnection.cc

```diff
@@ -273,8 +267,20 @@ bool PPTConnection::receive(map<string, string> &extensions, ostream *strm)
 	// if x then extensions follow, if d then data follows.
 	int bytesRead = readChunkHeader(_inBuff, 8);
 	BESDEBUG( MODULE, prolog << "Reading header, read " << bytesRead << " bytes" << endl );
-	if (bytesRead != 8)
-		throw BESInternalError(prolog + "Failed to read chunk header", __FILE__, __LINE__);
+    if (bytesRead == 0) {
+        INFO_LOG("PPTConnection::receive: read EOF from the OLFS, beslistener exiting.");
```

Comment:
This log line needs to be terminated with a _`\n`_

Reply:
Thanks.